From c668ce22b0dd72261621f19f124490eb5425eacd Mon Sep 17 00:00:00 2001
From: Matthew Smith <matthew@gentoo.org>
Date: Tue, 15 Nov 2022 21:38:50 +0000
Subject: [PATCH] Disable parallel build in Make 4.4

In versions of GNU Make prior to 4.4, ".NOTPARALLEL: compile" made the whole
build run serially. In 4.4 and later, ".NOTPARALLEL: compile" makes only
the compile target run serially.

This breaks the build and can lead to e.g. EEx being built before Mix.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index f6ba18d90e..8bb5bcdc65 100644
--- a/Makefile
+++ b/Makefile
@@ -24,7 +24,7 @@ SOURCE_DATE_EPOCH_PATH = lib/elixir/tmp/ebin_reproducible
 SOURCE_DATE_EPOCH_FILE = $(SOURCE_DATE_EPOCH_PATH)/SOURCE_DATE_EPOCH
 
 .PHONY: install compile erlang elixir unicode app build_plt clean_plt dialyze test check_reproducible clean clean_residual_files format install_man clean_man docs Docs.zip Precompiled.zip zips
-.NOTPARALLEL: compile
+.NOTPARALLEL:
 
 #==> Functions
 
